src/anim/evaluator/anim-evaluator.js

```diff
@@ -1,3 +1,130 @@
+import { Quat } from '../../math/quat.js';
+import { Vec3 } from '../../math/vec3.js';
+import { ANIM_LAYER_OVERWRITE } from '../controller/constants.js';
+
+class AnimTarget {
```

Comment:
just curious - why is AnimTarget class in anim-evaluator.js? I see in current version we already have anim-target.js in the same folder as well. Is it another AnimTarget class? Maybe pick a different name in that case.